### ISSUE TITLE
force_unwrap static function argument not getting lint error

### ISSUE THREAD

**Issue by mortenholmgaard** [realm/SwiftLint#3378]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

This line is not getting a force_unwrap lint error even thought it should:
```
Navigator.next(navigationController: self.navigationController!)

class Navigator {
    static func next(navigationController: UINavigationController) {
        ...
    }
}
```

It think it could have something todo with the fact that it is a static function but I am not sure. This is the only case it our codebase where this lint rule does not work.

### Environment

* SwiftLint version (run `swiftlint version` to be sure)?
0.40.3
* Installation method used (Homebrew, CocoaPods, building from source, etc)?
CocoaPods
* Paste your configuration file:

```yml
line_length: 150

excluded:
  - Pods
  - .idea

disabled_rules:
  - trailing_whitespace
  - vertical_parameter_alignment # all kinds of problems with this rule with false positives - multiple github issues

identifier_name:
  excluded:
    - id
    - to
    - i
  max_length: 50
type_name:
  excluded:
    - T
  max_length: 60 
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)?
No
* Which Xcode version are you using (check `xcodebuild -version`)?
Xcode 12.0.1
Build version 12A7300
* Do you have a sample that shows the issue? Run `echo "[string here]" | swiftlint lint --no-cache --use-stdin --enable-all-rules`
See description

**Comment by jpsim** [comment]
Can you share a more complete example that demonstrates the false negative? Something like this would be very helpful:

```
$ echo "[string here]" | swiftlint lint --no-cache --use-stdin --enable-all-rules
```

**Comment by mortenholmgaard** [comment]
This is very strange. I see that with the command it works just fine:
`$ echo "Navigator.next(navigationController: self.navigationController\!)" | swiftlint lint --no-cache --use-stdin --enable-all-rules
`

> <nopath>:1:58: warning: Force Unwrapping Violation: Force unwrapping should be avoided. (force_unwrapping)
> Done linting! Found 1 violation, 0 serious in 1 file.

But in this sample I continue to get no lint erros at all. I have tried with multiple cases all same result:
```
class Controller: UIViewController {
    private var testNavigationController: UINavigationController?

    override func viewDidLoad() {
        Navigator.next(navigationController: navigationController!)
        Navigator.next(navigationController: testNavigationController!)
    }
}


class Navigator {
    static func next(navigationController: UINavigationController) {
        ...
    }
}
```

**Comment by jpsim** [comment]
Those samples you posted all successfully report the force unwrapping violations for me:

```
$ cat file.swift 
class Controller: UIViewController {
    private var testNavigationController: UINavigationController?

    override func viewDidLoad() {
        Navigator.next(navigationController: navigationController!)
        Navigator.next(navigationController: testNavigationController!)
    }
}

class Navigator {
    static func next(navigationController: UINavigationController) {
        // ...
    }
}
$ swiftlint lint --path file.swift --enable-all-rules | grep force_unwrapping
Linting Swift files at paths file.swift
Linting 'file.swift' (1/1)
Done linting! Found 14 violations, 0 serious in 1 file.
/.../file.swift:5:66: warning: Force Unwrapping Violation: Force unwrapping should be avoided. (force_unwrapping)
/.../file.swift:6:70: warning: Force Unwrapping Violation: Force unwrapping should be avoided. (force_unwrapping)
$ swiftlint version                                  
0.41.0
```

**Comment by mortenholmgaard** [comment]
I just realized that force_unwrapping has been removed from our swiftlint.yml file - that is the reason - sorry.
